tests/commonTemplates_test.go

```diff
@@ -20,6 +24,8 @@ var _ = Describe("Common templates", func() {
 		editClusterRole testResource
 		goldenImageNS   testResource
 		testTemplate    testResource
+		currentVersion  = "v0.11.3"
```

Comment:
Instead of using this variable, the tests should use `Version` from `internal/operand/common-templates/resources.go`.